terraform/performance/main.tf

```diff
@@ -109,6 +109,13 @@ resource "null_resource" "validator_linux" {
   }
   provisioner "remote-exec" {
     inline = [
+      #mock server dependencies getting transfered.
+      "git clone --branch ${var.github_test_repo_branch} ${var.github_test_repo}",
+      "cd amazon-cloudwatch-agent-test && git checkout xray-performance-test", #@TODO remove before PR merge
```

Comment:
is this TODO relevant?

Reply:
yup! i am waiting for the PR to be ready before I remove it so I can test the code before the merge

Reply:
This should use the `github_test_repo_branch` var. You can specify it in your test workflow.